Source/AzureSqlBulkFailover.ps1

```diff
@@ -24,10 +35,25 @@ enum FailoverStatus {
     Failed # The failover process failed.
 }
 
-# helper function to log messages to the console including the date, name of the calling class and method
-function Log($message) {
-    $outputMessage = "$([DateTime]::Now.ToString("yyyy-MM-dd HH:mm:ss")) => $message"
-    Write-Verbose $outputMessage
+# Get the numeric value of the LogLevel to facilitate comparison
+function LogLevelValue($logLevel) {
+    switch ($logLevel) {
+        "Allways" { return 0; } # When passed to Log ensures the message is logged
+        "Minimal" { return 1; } # When defined as log level ensures only allways messages are logged
+        "Info" { return 2; } # When defined as log level ensures only info and allways messages are logged
+        "Verbose" { return 3; } # When defined as log level ensures all messages are logged
+        default { return 2; } # When LogLevel is not defined, default to Info

```

Comment:
Minor & probably ignorable: at line 20 the var is initialized to a default of 'Info', but here if the value is unrecognized it falls back on a default of 'Verbose'. Perhaps those defaults could be made consistent?